<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>970</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lnUsername">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>160</width>
     <height>50</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnPassword">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>470</y>
     <width>160</width>
     <height>50</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnNumber">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>540</y>
     <width>160</width>
     <height>50</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>SpaceShip Number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnModel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>400</y>
     <width>160</width>
     <height>50</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>SpaceShip Model</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnYear">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>470</y>
     <width>160</width>
     <height>50</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Production Year</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnColor">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>540</y>
     <width>160</width>
     <height>50</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>SpaceShip Color</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRegister">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>760</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>900</y>
     <width>96</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chckIs_VIP">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>640</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>VIP</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
